GPU/GPUTracking/Base/GPUReconstructionConvert.cxx

```diff
@@ -457,7 +457,7 @@ void zsEncoderRow::decodePage(std::vector<o2::tpc::Digit>& outputBuffer, const z
 
 #ifdef GPUCA_O2_LIB
 struct zsEncoderImprovedLinkBased : public zsEncoder {
-  static constexpr bool TIGHTLY_PACKED = false;
+  static constexpr bool TIGHTLY_PACKED = true;
```

Comment:
The improved link-based for now will only support TIGHTLY_PACKED = false.
Could you adapt the decoder to support both, and set this back to false?